app/src/main/AndroidManifest.xml

```diff
@@ -81,7 +81,7 @@
         <activity android:name=".activity.SettingsActivity"
             android:theme="@style/SettingsTheme"
             android:configChanges="locale"
-            android:windowSoftInputMode="adjustResize" />
+            android:windowSoftInputMode="adjustPan"/>
```

Comment:
Seeing that this was just added for #1660 - Was this purely for the Snackbars?

Reply:
I found this was a fix for the weird keyboard behavior - I assumed from the name and descriptions that adjustResize was the right flag but adjustPan fixed all the problems...